test/junit/scala/collection/immutable/LazyListTest.scala

```diff
@@ -349,13 +399,13 @@ class LazyListTest {
 
     // Check forced cyclic and non-cyclic streams
     assertEquals("LazyList(-2, -1)", pre(2).force.toString)
-    assertEquals("LazyList(0, 1, <cycle>)", cyc(2).force.toString)
-    assertEquals("LazyList(-2, -1, 0, 1, <cycle>)", precyc(2,2).force.toString)
+    assertEquals("LazyList(0, 1, 0, <cycle>)", cyc(2).force.toString)
```

Comment:
would it be possible and/or reasonable to have `<cycle>` also tell you the size of the cycle?

Reply:
relatedly, would it be reasonable to remove the separator before `<cycle>`?

Reply:
I'm pro comma, for the cycle size I have no opinion :)